app/javascript/components/widgets/weather/current-weather.test.jsx

```diff
@@ -27,12 +41,147 @@ describe('AnticipatedPrecipitation component', () => {
         precip.push({ precipitation: 0.0 });
       }
     }
-    const weather = { minutely: precip };
+    let byHour =
+      '[\n' +
+      '    {\n' +
+      '      "weather": {\n' +
+      '        "id": 804\n' +
+      '      }\n' +
+      '    },\n' +
+      '    {\n' +
+      '      "weather": {\n' +
+      '        "id": 804\n' +
```

Comment:
804 is clouds according to this:
https://openweathermap.org/weather-conditions#Weather-Condition-Codes-2

Should we be showing the precipitation message in this case?